packages/office-ui-fabric-react/src/components/Toggle/Toggle.base.tsx

```diff
@@ -78,7 +78,7 @@ export class ToggleBase extends BaseComponent<IToggleProps, IToggleState> implem
     });
 
     return (
-      <RootType className={classNames.root}>
+      <RootType className={classNames.root} hidden={(toggleNativeProps as any).hidden}>
```

Comment:
I know this is a bit much, but should we be doing {...toggleNativeProps} here to allow more than hidden?

Reply:
The PR description does specify that there is a discussion to be made whereas we should include other native props aside from hidden. My guess is that the decision would come down to what other native props would be passed and if we indeed want them to be passed down or not.

Reply:
My concern here is duplication of native props given that {...toggleNativeProps} is already present on the button. I assume we'd want to filter on onClick at least given that onClick is overridden on the button and shouldn't be present on the root element. Also not sure about aria- prefixed properties since those are passed into the KeytipData and button.

Reply:
Okay we can gradually adopt more if we need it.

Reply:
@khmakoto Do we also need this change for `Toggle` in experiments?

Reply:
We probably do.